optashift-employee-rostering-docs/src/main/asciidoc/Development/Server/Server-section.adoc

```diff
@@ -0,0 +1,5 @@
+[[serverDevelopment]]
+= Server Side Development of Optashift Employee Rostering
+:imagesdir: ../..
+
+TODO
```

Comment:
TODO's in documentation look bad and amateuristic. There are always documentation sections left to do. Docs are never perfect, never complete.

just leave it out the section are aren't documented yet (delete them). If they need to be tracked to do, create a jira.